WTWSMS/localisation/WtWSMS_custom_loc_strings.csv

```diff
@@ -0,0 +1,3 @@
+# Strings used in WtWSMS custom localisation;;;;;;;;;;;;x
+WtWSMS_string_eparchy;Eparchy;;;;;;;;;;;x
+WtWSMS_string_theme;Theme;;;;;;;;;;;x
```

Comment:
This could maybe be in the misc file, unless we add a lot of these.

Reply:
I plan to edit some other Roman titles like this (for example Dioceses being actually named Dioceses), so this file will be expanded.

Reply:
Ok, but on dioceses isn't the issue that we can't be consistent with the de jure and that it changes depending on culture rather than realms? By the way, here you use e_byzantium which makes sense, but what is displayed for a Greek realm which is outside of the ERE?

Reply:
Same as inside ERE, it still depends if Themata has been introduced.

Reply:
It doesn't necessarily make sense for it to suddenly change for a realm outside the ERE because the ERE adopts the theme system, although if a realm leaves the ERE after the adoption it makes sense for them to use the new name.

Reply:
I changed the approach, only Bureaucratic or Sub-Roman duchies should be affected now. Needs retesting, though.